style.css

```diff
@@ -80,4 +80,328 @@
+    font-size: 16px;
+    line-height: 24px;
+    padding-bottom: 24px;
+    color: #f4f5f7;
+  }
+
+  .button10,
+  .resume {
+    color: #fff;
+    background: #ff6b00;
+  }
+
+  .button10 {
+    height: 48px;
+    width: 161px;
+    border-style: none;
+  }
+
+  .footertext {
+    width: 327px;
+    height: 79px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 88px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+    margin-bottom: 24px;
+  }
+
+  .aboutmefooter {
+    background: #1c1a19;
+    background-image: url("./images/illustration-about-me-mobile.svg");
+    background-repeat: no-repeat;
+    padding: 130px 24px;
+    background-position: 100% 103%;
+  }
+
+  .languagesection {
+    background: #1c1a19;
+    color: #fff;
+    padding: 120px 24px;
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .languageskill1 {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .language {
+    height: 24px;
+    font-family: 'InaiMathi', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    margin-bottom: 13px;
+  }
+
+  .languagelist {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .languageitem {
+    padding: 0 12px;
+  }
+
+  .rectangle {
+    width: 18px;
+    height: 18px;
+  }
+
+  .item1,
+  .item2,
+  .item3,
+  .item4 {
+    padding: 0 12px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+  }
+
+  /* form part */
+  .formsection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 120px 24px;
+    gap: 58px;
+  }
+
+  .formtext {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .formcontainter {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+  }
+
+  input {
+    width: 327px;
+    height: 48px;
+    border-top: 0;
+    border-left: 0;
+    border-right: 0;
+    border-bottom: solid 0.6px #ccc;
+  }
+
+  textarea {
+    width: 327px;
+    height: 114px;
+    border: none;
+    background: #fbf8f7;
+    color: #3c3a39;
+    padding: 12px;
+  }
+
+  textarea::placeholder {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    color: #3c3a39;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .formitems {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 24px;
+    margin-bottom: 6rem;
+  }
+
+  .submitbutton {
```

Comment:
- [ ] [OPTIONAL] Make sure to style this button like the given Figma design.

## Yours:
![image](https://user-images.githubusercontent.com/92215878/192596022-e3e0c612-c1e9-4f27-bcd4-716f9bc79f41.png)
## Figma:
![image](https://user-images.githubusercontent.com/92215878/192596098-f06692b0-90cf-4b01-b622-f87599cff30f.png)